### ISSUE TITLE
support linking / rendering an example notebook?

### ISSUE THREAD

**Issue by cboettig** [radiantearth/stac-browser#310]
STAC-browser is really impressive, thanks for this fantastic project.

One thing we've noticed is that the browser prominently displays 'download' buttons for assets, which makes sense and all, but also can confuse our users when we want to encourage them to adopt more cloud-native workflows.  We really like the examples on Planetary Computer data catalog, where most datasets are accompanied with a [prominent "example notebook"](https://planetarycomputer.microsoft.com/dataset/landsat-c2-l2#Example-Notebook) tab, showing a Jupyter notebook with code and supporting text that walks a user through basic access.  It looks like we can hack something like this into the `description` metadata (which appears to support markdown, wonderful thank you!!), but would love some more intuitive way to integrate or at least more prominently link to example notebooks.  (Ideally a mechanism that could support notebooks in various languages, e.g. Jupyter, quarto/Rmd, etc).

**Comment by m-mohr** [comment]
Yeah, ideally this would resolve in an extension that STAC Browser could implement. 
Back in 2021 we had a discussion around example code (which a notebook basically is), you can find it here: https://github.com/radiantearth/stac-spec/issues/992 (example: https://github.com/radiantearth/stac-spec/issues/992#issuecomment-812591806 )
It never evolved into an extension though so there's no explicit support in STAC Browser yet.

**Comment by cboettig** [comment]
@m-mohr thanks, that's a great thread; I'll follow up there.  Yes, using the links field with an appropriate rel type does sound like a reasonable solution.

**Comment by m-mohr** [comment]
This seems related to #114. If we add a section anyway where code examples will be shown for the libraries, we can also add the "examples" links there.

**Comment by m-mohr** [comment]
Proposal for an extension: https://github.com/stac-extensions/example-links
